app.json

```diff
@@ -34,7 +34,7 @@
     "android": {
       "googleServicesFile": "./google-services.json",
       "package": "org.timeoverflow.mobileapp",
-      "versionCode": 20
+      "versionCode": 22
```

Comment:
looks like we could get Expo to this increment for us if we specified `"autoIncrement": true` in the production profile in eas.json. See _remote version source_ on https://docs.expo.dev/build-reference/app-versions/.

I'm not meaning to do it here, but it's some worth exploring soon. This version bump is to me the biggest bottleneck to publish versions and automate the process.